provisioning_templates/provision/kickstart_default.erb

```diff
@@ -40,7 +41,7 @@ This template accepts the following parameters:
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
-<% if @host.operatingsystem.name != 'Fedora' || os_major < 29 -%>
+<% if (is_fedora && os_major >= 28) || os_major >= 8 -%>
```

Comment:
You reversed the logic here. In other places we also add a check `rhel_compatible` when comparing os majors so that you don't break older Fedora versions.

Reply:
What did I reverse here? It looks good to me assuming that I will add the missing `rhel_compatible`.

Reply:
Oh I get it it's the `install` command which was *removed*.

Reply:
Correct. I'll admit my original code wasn't the clearest.